miRge.pl

```diff
@@ -36,7 +36,7 @@
 my $version = "2.0";
 local $ENV{PATH} = "$ENV{PATH}:".$miRgePath;
 $miRgePath =~ s/\/[^\/]+\.pl/\//;
-my $refPath = $miRgePath."miRge.seqLibs/";
+my $refPath = $miRgePath."usr/local/miRge/miRge.seqLibs/";
```

Comment:
This change will result in some funky things. `miRgePath` is set to the directory of miRge.pl so I don't see how this is compatible. What caused you to need this change?